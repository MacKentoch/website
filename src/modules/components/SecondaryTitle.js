import styled from 'styled-components'
import theme from 'style/theme'

const SecondaryTitle = styled.h2`
  font-size: 26px;
  line-height: 1.4;
  font-weight: 300;
  margin: 10px 0;

  @media (min-width: ${theme.medias.phablet}) {
    font-size: 40px;
  }
`

export default SecondaryTitle
