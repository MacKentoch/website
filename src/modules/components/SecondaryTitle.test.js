import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SecondaryTitle from './SecondaryTitle'

describe('SecondaryTitle', () => {
  it('should render a h2', () => {
    const wrapper = shallow(<SecondaryTitle>Hello</SecondaryTitle>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
