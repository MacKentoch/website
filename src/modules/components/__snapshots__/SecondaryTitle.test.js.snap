// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryTitle should render a h2 1`] = `
<h2
  className="SecondaryTitle-s1o6tp06-0 irQbez"
>
  Hello
</h2>
`;
